packages/terra-table/src/Table.scss

```diff
@@ -13,133 +14,75 @@
   }
 
   tr {
-    display: block;
+    display: table-row;
+    margin-bottom: 0;
   }
 
   th,
   td {
     @include terra-text-align-start;
+    border: 0;
+    border-left: $terra-table-cell-border;
```

Comment:
We want to use the bidi mixins instead of directional CSS properties for things like `border-left`.
However, [the mixin for border-left](https://github.com/cerner/terra-core/tree/master/packages/terra-mixins/docs#terra-border-start--terra-border-end) currently expects a format like `@include terra-border-start ($width, $style, $color)`.

`$terra-table-cell-border` is defined as a single string. We'll want to update the `terra-border-start` and `terra-border-end` mixins to take a single string to match this.